src/main/java/edu/harvard/iq/dataverse/harvest/server/xoai/DataverseXoaiItem.java

```diff
@@ -1,40 +1,37 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
 package edu.harvard.iq.dataverse.harvest.server.xoai;
 
-import com.lyncode.xoai.dataprovider.model.Item;
-import com.lyncode.xoai.dataprovider.model.Set;
-import com.lyncode.xoai.model.oaipmh.About;
+import io.gdcc.xoai.dataprovider.model.Item;
+import io.gdcc.xoai.dataprovider.model.Set;
+import io.gdcc.xoai.model.oaipmh.results.record.Metadata;
+import io.gdcc.xoai.model.oaipmh.results.record.About;
 import edu.harvard.iq.dataverse.Dataset;
 import edu.harvard.iq.dataverse.harvest.server.OAIRecord;
 import edu.harvard.iq.dataverse.util.StringUtil;
+import java.time.Instant;
 import java.util.ArrayList;
-import java.util.Date;
 import java.util.List;
 
 
 /**
  *
  * @author Leonid Andreev
  * 
- * This is an implemention of an Lyncode XOAI Item; 
+ * This is an implemention of a Lyncode/DSpace/gdcc XOAI Item.
  * You can think of it as an XOAI Item wrapper around the
  * Dataverse OAIRecord entity.
  */
-public class Xitem implements Item {
+public class DataverseXoaiItem implements Item {
```

Comment:
**nitpick:** shouldn't this class be `final`? (Yes, I'm a huge fan of non-modifiable by default)